packages/backend/app/Controllers/Http/ApartmentsController.ts

```diff
@@ -41,6 +41,7 @@ export default class ApartmentsController {
         .preload('banners')
         .preload('photos')
         .preload('reviews')
+        .preload('buildingType')
```

Comment:
```suggestion
        .preload('buildingTypes')
```
Раз их несколько может быть.

Reply:
ага